templates/oth/index.html

```diff
@@ -177,11 +177,11 @@ <h3>Rules</h3>
                     <p data-scroll-reveal="enter top and move 20px over 1.4s" class="days_ref">
                         days</p>
                 </li>
-                <li><span data-scroll-reveal="enter bottom and move 20px over 1.6s" class="hours">00</span>
+                <li><span data-scroll-reveal="enter bottom and move 20px over 1.6s" class="hours">2</span>
                     <p data-scroll-reveal="enter top and move 20px over 1.8s" class="hours_ref">
                         hours</p>
                 </li>
-                <li><span data-scroll-reveal="enter bottom and move 2px over 2.0s" class="minutes">00</span>
+                <li><span data-scroll-reveal="enter bottom and move 2px over 2.0s" class="minutes">20</span>
                     <p data-scroll-reveal="enter top and move 20px over 2.2s" class="minutes_ref">
```

Comment:
is this for the timer?

Reply:
Make it zero. The changes are in javascript, not here.

Reply:
Yes